frontend/public/components/_edit-yaml.scss

```diff
@@ -32,6 +32,26 @@
   margin: 0;
 }
 
+.yaml-editor__link {
+  display: inline-block;
+}
+
+.yaml-editor__links {
+  position: absolute;
+  top: 5px;
+  right: 0;
+  z-index: 1;
+  @media(max-width: $screen-sm-max) {
+    display: none;
+  }
+}
+
+.yaml-editor__links-divider {
```

Comment:
Since this PR refuses to merge and I overlooked it the first time...

Please update to use `.co-action-divider` as that will make this consistent with `resource-log.jsx`

Reply:
It looks like .co-action-divider doesn't have any padding; do you suggest adding it to the links and removing the padding conditionally?

Reply:
I think it probably makes sense to add a modifier to `.co-action-divider` (e.g., `.co-action-divider--spaced`) to add any necessary declarations.